packages/server/utils/stripe/StripeManager.ts

```diff
@@ -54,19 +75,14 @@ export default class StripeManager {
     })
   }
 
-  async createSetupIntent(customerId: string) {
-    return this.stripe.setupIntents.create({
-      customer: customerId
-    })
-  }
-
   async createTeamSubscription(customerId: string, orgId: string, quantity: number) {
     return this.stripe.subscriptions.create({
       // USE THIS FOR TESTING A FAILING PAYMENT
       // https://stripe.com/docs/billing/testing
       // trial_end: toEpochSeconds(new Date(Date.now() + 1000 * 10)),
       customer: customerId,
       proration_behavior: 'none',
+      payment_behavior: 'default_incomplete',
```

Comment:
We need this for 3D Secure cards so that the payment is considered incomplete before the user authorises it